src/workers/race_event_status_worker.rs

```diff
@@ -72,11 +89,156 @@ enum RaceEventContentAndStatus {
 struct RaceInfoBundle {
     race: BracketRace,
     bri: BracketRaceInfo,
-    bracket: Bracket,
+    status: RaceEventContentAndStatus,
+}
+
+struct GuildEventConfig {
+    guild_id: Id<GuildMarker>,
+    #[allow(unused)]
+    // very useful in debugging but not necessarily logged outside of development, so allow unused
+    guild_name: String,
+}
+
+impl GuildEventConfig {
+    /// is this guild interested in tracking this race?
+    fn should_sync_race(&self, _race: &BracketRace) -> bool {
```

Comment:
i started writing scaffolding for user-configurable event syncing but then decided to split that work out, so there's a couple things that are still stubs

Reply:
see #165 for this